src/test/java/com/commercetools/sync/products/utils/ProductVariantUpdateActionUtilsTest.java

```diff
@@ -5,37 +5,73 @@
 import io.sphere.sdk.products.commands.updateactions.SetSku;
 import org.junit.Test;
 
-import static com.commercetools.sync.products.utils.ProductVariantUpdateActionUtils.buildProductVariantSkuUpdateActions;
+import static com.commercetools.sync.products.utils.ProductVariantUpdateActionUtils.buildProductVariantSkuUpdateAction;
 import static org.assertj.core.api.Assertions.assertThat;
 import static org.mockito.Mockito.mock;
 import static org.mockito.Mockito.when;
 
 public class ProductVariantUpdateActionUtilsTest {
 
     @Test
-    public void buildProductVariantSkuUpdateActions_normalCase() throws Exception {
+    public void buildProductVariantSkuUpdateAction_WithBothNullSkus_ShouldNotBuildAction() {
         final ProductVariant variantOld = mock(ProductVariant.class);
         final ProductVariantDraft variantDraftNew = mock(ProductVariantDraft.class);
 
-        assertThat(buildProductVariantSkuUpdateActions(variantOld, variantDraftNew)).isEmpty();
+        assertThat(buildProductVariantSkuUpdateAction(variantOld, variantDraftNew)).isEmpty();
+    }
+
+    @Test
+    public void buildProductVariantSkuUpdateAction_WithNullNewSku_ShouldBuildUpdateAction() {
+        final ProductVariant variantOld = mock(ProductVariant.class);
+        final ProductVariantDraft variantDraftNew = mock(ProductVariantDraft.class);
 
         when(variantOld.getSku()).thenReturn("sku-old");
         when(variantOld.getId()).thenReturn(42);
-        assertThat(buildProductVariantSkuUpdateActions(variantOld, variantDraftNew))
-                .containsExactly(SetSku.of(42, null, true));
 
+        final SetSku updateAction =
+            buildProductVariantSkuUpdateAction(variantOld, variantDraftNew).orElse(null);
+        assertThat(updateAction).isNotNull();
+        assertThat(updateAction).isEqualTo(SetSku.of(42, null, true));
```

Comment:
_AssertJ_ has special assert for optionals:
```java
assertThat(buildProductVariantSkuUpdateAction(variantOld, variantDraftNew)).contains(SetSku.of(42, null, true));
```

and same below

Reply:
thanks :)

Reply:
f1363ce